src/objective-c/CronetFramework.podspec

```diff
@@ -69,7 +69,7 @@ Pod::Spec.new do |s|
   s.vendored_framework = "Cronet.framework"
   s.author             = "The Chromium Authors"
   s.ios.deployment_target = "8.0"
-  s.source       = { :http => 'https://storage.googleapis.com/grpc-precompiled-binaries/cronet/Cronet.framework.zip' }
+  s.source       = { :http => 'https://storage.googleapis.com/grpc-precompiled-binaries/cronet/Cronet.framework-1.0.0.zip' }
```

Comment:
I would put the version in a variable `v`, and use it both for `s.version` and here.